etna/analysis/plotters.py

```diff
@@ -1159,10 +1191,18 @@ def plot_feature_relevance(
                     f"Relevances on segment: {segment} of features: {na_relevance_features} can't be calculated."
                 )
             relevance = relevance.dropna()[:top_k]
-            if normalized:
-                relevance = relevance / relevance.sum()
-            sns.barplot(x=relevance.values, y=relevance.index, orient="h", ax=ax[i])
-            ax[i].set_title(f"Feature relevance: {segment}")
+            if type(relevance_table) is StatisticsRelevanceTable:
```

Comment:
Checking type is bad, better to check `isinstance(relevance_table, StatisticsRelevanceTable)`